kalite/store/models.py

```diff
@@ -98,12 +97,11 @@ def handle_exam_unset(sender, **kwargs):
         for testlog in testlogs:
```

Comment:
In fact, can probably delete this entire for loop, as we are not awarding points for tests.